proto/commonfate/control/log/v1alpha1/authz_eval.proto

```diff
@@ -2,119 +2,112 @@ syntax = "proto3";
+  }
 }
 
 message QueryEvaluationsRequest {
-    // The token for the next page.
-    string page_token = 1;
+  // The token for the next page.
+  string page_token = 1;
 
-    // Filters for the query. Events matching *all* filters will be returned.
-    repeated Filter filters = 2;
+  // Filters for the query. Events matching *all* filters will be returned.
+  repeated Filter filters = 2;
 }
 
 // Filters events with a particular authorization decision
 message DecisionFilter {
-    commonfate.authz.v1alpha1.Decision decision = 1;
+  commonfate.authz.v1alpha1.Decision decision = 1;
 }
 
 // Filters event by a particular key/value tag associated when them.
 // Will filter for either events which match the tag, or do not match the tag,
 // based on the 'comparison' value.
 message TagFilter {
-    string key = 1;
-    string value = 2;
-    BoolComparison comparison = 3;
+  string key = 1;
+  string value = 2;
+  BoolComparison comparison = 3;
 }
 
 // Filters events where the principal/action/resource matches particular entities.
 message EntityFilter {
-    // IDs of entities to filter for. Will filter for a principal/action/resource matching any of these entities.
-    repeated commonfate.entity.v1alpha1.EID ids = 1;
+  // IDs of entities to filter for. Will filter for a principal/action/resource matching any of these entities.
+  repeated commonfate.entity.v1alpha1.EID ids = 1;
 }
 
 // Filters events based on when they occurred.
 message OccurredAtFilter {
-    google.protobuf.Timestamp time = 1;
-    TimeComparison comparison = 2;
+  google.protobuf.Timestamp time = 1;
+  TimeComparison comparison = 2;
 }
 
 message Filter {
-    oneof filter {
-        TagFilter tag = 1;
-        OccurredAtFilter occurred_at = 2;
-        EntityFilter principal = 3;
-        EntityFilter action = 4;
-        EntityFilter resource = 5;
-        DecisionFilter decision = 6;
-    }
+  oneof filter {
+    TagFilter tag = 1;
+    OccurredAtFilter occurred_at = 2;
+    EntityFilter principal = 3;
+    EntityFilter action = 4;
+    EntityFilter resource = 5;
+    DecisionFilter decision = 6;
+  }
 }
 
 enum TimeComparison {
-    TIME_COMPARISON_UNSPECIFIED = 0;
+  TIME_COMPARISON_UNSPECIFIED = 0;
 
-    // Events which occurred before the specified time.
-    TIME_COMPARISON_BEFORE = 1;
+  // Events which occurred before the specified time.
+  TIME_COMPARISON_BEFORE = 1;
 
-    // Events which occurred after the specified time.
-    TIME_COMPARISON_AFTER = 2;
+  // Events which occurred after the specified time.
+  TIME_COMPARISON_AFTER = 2;
 }
 
 enum BoolComparison {
-    BOOL_COMPARISON_UNSPECIFIED = 0;
+  BOOL_COMPARISON_UNSPECIFIED = 0;
 
-    // Equal to the provided value.
-    BOOL_COMPARISON_EQUAL = 1;
+  // Equal to the provided value.
+  BOOL_COMPARISON_EQUAL = 1;
 
-    // Not equal to the provided value.
-    TIME_COMPARISON_NOT_EQUAL = 2;
+  // Not equal to the provided value.
+  TIME_COMPARISON_NOT_EQUAL = 2;
```

Comment:
I noticed that this is meant to be `BOOL_COMPARISON_NOT_EQUAL`
```suggestion
  TIME_COMPARISON_NOT_EQUAL = 2;
```

Reply:
I'll raise a separate PR for this, I think the field rename should not be a breaking change